cmd/snap/cmd_snap_op.go

```diff
@@ -556,7 +556,7 @@ func (x *cmdInstall) installOne(nameOrPath, desiredName string, opts *client.Sna
 }
 
 func isLocalSnap(name string) bool {
-	return strings.Contains(name, "/") || strings.HasSuffix(name, ".snap") || strings.Contains(name, ".snap.")
+	return strings.Contains(name, "/") || strings.Contains(name, ".")
```

Comment:
this change would make things more consistent but OTOH we have lived with the current logic as is for a long time. And this could affect on how we can evolve the syntax used for modifiers of non-local snap installations, which so far has  started using "_" for parallel installs, and it will start using "+" for components, so from that POV I would prefer to leave things as is.